libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -1237,6 +1240,8 @@ def output() -> Iterator:
                 config[CONF][CONFIG_KEY_STREAM_WRITER] = lambda c: stream.put(
                     ((), "custom", c)
                 )
+            if self.store:
```

Comment:
this is the acutal change.

Very open to other suggestions.

Issue was that the store wasn't being set anymore now that we aren't necessarily deifning managed channels

Reply:
Ah